task/tekton-catalog-publish/0.1/README.md

```diff
@@ -0,0 +1,82 @@
+# Tekton Catalog Publish
+
+This task publishes each Task in a Tekton catalog as [Tekton Bundles](https://tekton.dev/docs/pipelines/tekton-bundle-contracts/).
```

Comment:
Does the user of this task really risk running afoul of the contract? If not, https://tekton.dev/docs/pipelines/pipelines/#tekton-bundles is probably a better link since it explains why you would want to use bundles.

Reply:
good catch, thank you